api/types/events/oneof.go

```diff
@@ -704,6 +704,14 @@ func ToOneOf(in AuditEvent) (*OneOf, error) {
 		out.Event = &OneOf_IntegrationDelete{
 			IntegrationDelete: e,
 		}
```

Comment:
I think you're missing an entry in `lib/events/dynamic.go` as well

Reply:
76e04ba15c